src/models/StyleSheet.js

```diff
@@ -3,6 +3,28 @@
 /* Wraps glamor's stylesheet and exports a singleton for the rest
 *  of the app to use. */
 
-import { StyleSheet } from '../vendor/glamor/sheet'
+import { StyleSheet as GlamorStyle } from '../vendor/glamor/sheet'
 
-export default new StyleSheet({ speedy: false, maxLength: 40 })
+class StyleSheet {
+  styleSheet: any
```

Comment:
@relekang also reading the flow documentation it seems that typeof statement in `import` is actually short ciruciting `as` which is a valid es2015 statement.  It's inferring it as type instead of a variable.